Flaky run of test_regression_unfiltered_route

## Problem Definition

```
=================================== FAILURES ===================================
____________ test_regression_unfiltered_routes[matrix-False-64-0-5] ____________

kwargs = {'deposit': 200, 'raiden_network': [<App 0xc5fBAd78dAB738bd9EaCF7aF5D00Dc4190964B95>, <App 0xF80597f0666e447799Db9445b...b0061344E>], 'settle_timeout': 64, 'token_addresses': [b':\xe2R\x0c&y\x94\xde\x0c\x95\xacbN\xd2\xfe\x9e\xa4\x87\xd6E']}
result = <gevent._event.AsyncResult object at 0x7fc26ed7c130>
apps = [<App 0xc5fBAd78dAB738bd9EaCF7aF5D00Dc4190964B95>, <App 0xF80597f0666e447799Db9445b0C237E9D585F906>, <App 0x9C5186D5Ce...Dd2FCF8D3CECE6676>, <App 0xD8782e2D1AC823128068ECC5118Fda377E65Dc90>, <App 0x1bB9C2C9F3F1B8e2124c09F928eD0c8b0061344E>]
app = <App 0x1bB9C2C9F3F1B8e2124c09F928eD0c8b0061344E>
test_greenlet = <Greenlet "Greenlet-10" at 0x7fc26d7dd5f0: test_regression_unfiltered_routes(raiden_network=[<App 0xc5fBAd78dAB738bd9E...D00Dc4190964B95>,, token_addresses=[b':\xe2R\x0c&y\x94\xde\x0c\x95\xacbN\xd2\xfe\x9e\, settle_timeout=64, deposit=200)>

    @wraps(test_function)
    def wrapper(**kwargs):
        result = AsyncResult()
        apps = kwargs.get("raiden_network", kwargs.get("raiden_chain"))
        if not apps:
            raise Exception(
                f"Can't use `raise_on_failure` on test function {test_function.__name__} "
                "which uses neither `raiden_network` nor `raiden_chain` fixtures."
            )
    
        # Do not use `link` or `link_value`, an app an be stopped to test restarts.
        for app in apps:
            assert app.raiden, "The RaidenService must be started"
            app.raiden.link_exception(result)
    
        test_greenlet = gevent.spawn(test_function, **kwargs)
        test_greenlet.link(result)
    
        # Returns if either happens:
        # - The test finished (successfully or not)
        # - One of the apps crashed during the test
        try:
>           result.get()

raiden/tests/utils/detect_failure.py:40: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
src/gevent/event.py:381: in gevent._event.AsyncResult.get
    ???
src/gevent/event.py:409: in gevent._event.AsyncResult.get
    ???
src/gevent/event.py:399: in gevent._event.AsyncResult.get
    ???
src/gevent/event.py:379: in gevent._event.AsyncResult._raise_exception
    ???
../venv-3.7-LINUX/lib/python3.7/site-packages/gevent/_compat.py:47: in reraise
    raise value.with_traceback(tb)
src/gevent/greenlet.py:716: in gevent._greenlet.Greenlet.run
    ???
raiden/raiden_service.py:446: in _run
    self.stop_event.wait()
src/gevent/event.py:240: in gevent._event.Event.wait
    ???
src/gevent/event.py:140: in gevent._event._AbstractLinkable._wait
    ???
src/gevent/event.py:117: in gevent._event._AbstractLinkable._wait_core
    ???
src/gevent/event.py:119: in gevent._event._AbstractLinkable._wait_core
    ???
src/gevent/_greenlet_primitives.py:59: in gevent.__greenlet_primitives.SwitchOutGreenletWithLoop.switch
    ???
src/gevent/_greenlet_primitives.py:59: in gevent.__greenlet_primitives.SwitchOutGreenletWithLoop.switch
    ???
src/gevent/_greenlet_primitives.py:63: in gevent.__greenlet_primitives.SwitchOutGreenletWithLoop.switch
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   RuntimeError: dictionary changed size during iteration

src/gevent/__greenlet_primitives.pxd:35: RuntimeError
```


The full logs are here: https://gist.github.com/LefterisJP/69bda70c9e6aeff79a4516f823a8b733

--- a/raiden/tests/integration/long_running/test_stress.py
+++ b/raiden/tests/integration/long_running/test_stress.py
@@ -364,7 +364,7 @@ def test_stress(
 ) -> None:
     token_address = token_addresses[0]
     rest_apis = start_apiserver_for_network(raiden_network, port_generator)
-    identifier_generator = count()
+    identifier_generator = count(start=1)
 
     token_network_address = views.get_token_network_address_by_token_address(
         views.state_from_app(raiden_network[0]),

--- a/raiden/tests/integration/test_recovery.py
+++ b/raiden/tests/integration/test_recovery.py
@@ -29,7 +29,6 @@ from raiden.utils.transfers import create_default_identifier
 from raiden.utils.typing import BlockNumber, PaymentAmount, PaymentID
 
 
-@pytest.mark.skip(reason="flaky, see https://github.com/raiden-network/raiden/issues/5133")
 @raise_on_failure
 @pytest.mark.parametrize("deposit", [10])
 @pytest.mark.parametrize("channels_per_node", [CHAIN])

--- a/raiden/tests/integration/test_regression.py
+++ b/raiden/tests/integration/test_regression.py
@@ -53,7 +53,6 @@ def open_and_wait_for_channels(app_channels, registry_address, token, deposit, s
     wait_for_channels(app_channels, registry_address, [token], deposit)
 
 
-@pytest.mark.skip(reason="flaky, see https://github.com/raiden-network/raiden/issues/5195")
 @raise_on_failure
 @pytest.mark.parametrize("number_of_nodes", [5])
 @pytest.mark.parametrize("channels_per_node", [0])

--- a/raiden/tests/integration/test_send_queued_messages.py
+++ b/raiden/tests/integration/test_send_queued_messages.py
@@ -24,7 +24,6 @@ from raiden.utils.typing import BlockNumber, PaymentID, TokenAmount
 @pytest.mark.parametrize("deposit", [10])
 @pytest.mark.parametrize("channels_per_node", [CHAIN])
 @pytest.mark.parametrize("number_of_nodes", [2])
-@pytest.mark.skip("Issue: #4312")
 def test_send_queued_messages(  # pylint: disable=unused-argument
     raiden_network, deposit, token_addresses, network_wait
 ):

--- a/raiden/tests/integration/transfer/test_mediatedtransfer.py
+++ b/raiden/tests/integration/transfer/test_mediatedtransfer.py
@@ -245,7 +245,7 @@ def test_mediated_transfer_with_entire_deposit(
         )
 
 
-@pytest.mark.skip(reason="flaky, see https://github.com/raiden-network/raiden/issues/4804")
+@pytest.mark.skip(reason="flaky, see https://github.com/raiden-network/raiden/issues/4694")
 @raise_on_failure
 @pytest.mark.parametrize("channels_per_node", [CHAIN])
 @pytest.mark.parametrize("number_of_nodes", [3])

